src/main/java/org/prnhs/javaee/swim/dto/PlanResultDto.java

```diff
@@ -1,15 +1,23 @@
 package org.prnhs.javaee.swim.dto;
 
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
 import java.sql.Time;
```

Comment:
Looks like i missed this in previous PR, don't use java.sql.Time here since it's core centric.  Find another one or simply use the Date object.  Or you can try the new LocalDateTIme or whatever was included in latest java releases.